### ISSUE TITLE
Please add leadid_id to integration

### ISSUE THREAD

**Issue by kcaponeAP** [activeprospect/leadconduit-integration-lowermybills#30]
LowerMyBills is now sending leadid_id with their leads for Bridgeview.

Can we please have this routed to the LCN standard field: leadid_id

sample lead: https://next.leadconduit.com/events/55a6996b281d062a2f08b3f5?source_id=535e9fdd94149d05b5000003&type=source&start=2015-07-15&end=2015-07-15

@alexkwolfe can you please relate the downcase issue to this for priority from the LCN meeeting

```xml
<LeadiD_ID>D2EA4227-3C88-010D-3B72-AAF210CBE8CF</LeadiD_ID>
```

**Comment by alexkwolfe** [comment]
If I understand correctly, the downcast issue only applies to the generic post integration.

**Comment by cgrayson** [comment]
Fixed in 1.2.5.
